content/en/community/end-user/interviews-feedback.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Interviews or Feedback Sessions
+linkTitle: Interviews or Feedback Sessions
```

Comment:
Drop the `linkTitle` if it's the same as the `title`, or maybe shorten it like this:

```suggestion
linkTitle: Interviews or Feedback
```

WDYT?

Reply:
I went ahead and dropped the `linkTitle`. Thank you!